views/samples/contents.hbs

```diff
@@ -76,15 +75,19 @@
 			});
 		});
 
-		$('#main-nav', document).css({
-			'margin-top': $('#top-nav', document).height() + 30,
-			'margin-bottom': '30px'
+		$('#main-nav', document).css({ 'top': Math.max(120, $('#top-nav', document).height() + 20) });
```

Comment:
Does it mean that 120 is the height of the header or something different?

Reply:
Yes, the fixed side nav is at least 120 pixel in height. Somehow $().height is not always reliable.